Mon Sep 12 20:10:18 2022 - INFO  - Start to prepare the test environment.
../common/common_podman.sh: line 21: /etc/containers/registries.conf: No such file or directory
oe_test_podman_DK_11.sh: line 24: podman: command not found
oe_test_podman_DK_11.sh: line 25: podman: command not found
Mon Sep 12 20:11:08 2022 - INFO  - End to prepare the test environment.
Mon Sep 12 20:11:08 2022 - INFO  - Start to run test.
oe_test_podman_DK_11.sh: line 31: docker: command not found
oe_test_podman_DK_11.sh: line 32: docker: command not found
Mon Sep 12 20:11:09 2022 - ERROR - oe_test_podman_DK_11.sh line 33
oe_test_podman_DK_11.sh: line 34: docker: command not found
oe_test_podman_DK_11.sh: line 35: docker: command not found
Mon Sep 12 20:11:10 2022 - ERROR - oe_test_podman_DK_11.sh line 36
oe_test_podman_DK_11.sh: line 37: docker: command not found
oe_test_podman_DK_11.sh: line 38: docker: command not found
Mon Sep 12 20:11:10 2022 - ERROR - oe_test_podman_DK_11.sh line 39
oe_test_podman_DK_11.sh: line 40: docker: command not found
oe_test_podman_DK_11.sh: line 41: docker: command not found
Mon Sep 12 20:11:11 2022 - ERROR - oe_test_podman_DK_11.sh line 42
oe_test_podman_DK_11.sh: line 44: docker: command not found
oe_test_podman_DK_11.sh: line 45: docker: command not found
Mon Sep 12 20:11:12 2022 - ERROR - oe_test_podman_DK_11.sh line 46
oe_test_podman_DK_11.sh: line 47: docker: command not found
oe_test_podman_DK_11.sh: line 48: docker: command not found
Mon Sep 12 20:11:13 2022 - ERROR - oe_test_podman_DK_11.sh line 49
oe_test_podman_DK_11.sh: line 50: docker: command not found
oe_test_podman_DK_11.sh: line 51: docker: command not found
Mon Sep 12 20:11:13 2022 - ERROR - oe_test_podman_DK_11.sh line 52
oe_test_podman_DK_11.sh: line 53: docker: command not found
oe_test_podman_DK_11.sh: line 54: docker: command not found
Mon Sep 12 20:11:14 2022 - ERROR - oe_test_podman_DK_11.sh line 55
Mon Sep 12 20:11:15 2022 - INFO  - End to run test.
Mon Sep 12 20:11:15 2022 - ERROR - failed to execute the case.
Mon Sep 12 20:11:16 2022 - INFO  - Start to restore the test environment.
../common/common_podman.sh: line 33: podman: command not found
../common/common_podman.sh: line 34: podman: command not found
../common/common_podman.sh: line 35: podman: command not found
Traceback (most recent call last):
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 1:15:01 ago on Mon 12 Sep 2022 06:56:05 PM CST.\nError: \n Problem 1: conflicting requests\n  - nothing provides crun needed by podman-1:3.4.4-2.oe2209.riscv64\n Problem 2: package podman-docker-1:3.4.4-2.oe2209.noarch requires podman = 1:3.4.4-2.oe2209, but none of the providers can be installed\n  - conflicting requests\n  - nothing provides crun needed by podman-1:3.4.4-2.oe2209.riscv64\n(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)"
Mon Sep 12 20:11:19 2022 - INFO  - End to restore the test environment.
